docs/_config.yml

```diff
@@ -1,5 +1,5 @@
 baseurl: /
-exclude: design
+exclude: [design]
```

Comment:
I had to make this change in order for `jekyll` to run correctly locally. I installed the latest Ruby (`3.3.4`) and latest Jekyll (`4.3.3`) on my machine and attempting to run Jekyll caused this error: 

```
Configuration file: /Users/sean/repos/rq/docs/_config.yml
jekyll 4.3.3 | Error:  'exclude' should be set as an array, but was: "design".
/Users/sean/.gem/ruby/3.3.4/gems/jekyll-4.3.3/lib/jekyll/configuration.rb:291:in `block in check_include_exclude': 'exclude' should be set as an array, but was: "design". (Jekyll::Errors::InvalidConfigurationError)
```